### ISSUE TITLE
Feature Request - Spell Effect Clickable Defense

### ISSUE THREAD

**Issue by 7H3LaughingMan** [shemetz/pf2e-extempore-effects#42]
When generating an effect from a spell would it be possible to make the "Defense" part at the top a clickable link with the relevant spellcasting DC?

Before
![Before](https://github.com/user-attachments/assets/e0337637-1163-4030-91fd-68bb6b3f256b)

After
![After](https://github.com/user-attachments/assets/0499371f-d3f1-49b6-bb86-554e9eb2e727)

**Comment by shemetz** [comment]
Yeah, this makes sense!  I think it can be done pretty easily, I'll try that.

**Comment by shemetz** [comment]
Fixed in v1.8.7
